src/machine/instruction.cpp

```diff
@@ -1514,3 +1514,133 @@ TokenizedInstruction::TokenizedInstruction(
     , address(address)
     , filename(std::move(filename))
     , line(line) {}
+
+
+void gen_instruction_code_and_expected_str(QTextStream& outfile, const InstructionMap* im, uint32_t code, int count, int offset, int subclass_i, int depth) {
```

Comment:
This should be in the instruction.test.cpp file. It has no use during normal operation of the simulator. Also I would prefer the name "generate_code_and_string_data" and a little docstring explaining the usage.

Reply:
agree